RtAudio.cpp

```diff
@@ -47,6 +47,7 @@
 #include <cstring>
 #include <climits>
 #include <cmath>
+#include <cstring>
```

Comment:
This is not necessary since the hack in the first commit is no longer present and the openStream function always have a updated buffersize already.

Reply:
If it's not necessary, why is it still here?